### PR TITLE
Add MAXIMUM_MEMORY to 4GB and Rename TOTAL_MEMORY to INITIAL_MEMORY

### PR DESCRIPTION
**Pull request by airen1986**
### Description

As per blog: https://v8.dev/blog/4gb-wasm-memory , WASM memory could be max 4GB in Chrome V8 based browser. Also param "TOTAL_MEMORY" is update to "INITIAL_MEMORY"

### CODE REVIEW

**Comment by hoodmane** [comment]
Thanks @airen1986. What's the difference between TOTAL_MEMORY and INITIAL_MEMORY? Are they synonyms?

**Comment by airen1986** [comment]
Yes, both are synonyms. INITIAL_MEMORY is latest name, you can check this documentation below from EMSCRIPTEN: https://emscripten.org/docs/tools_reference/settings_reference.html?highlight=environment#initial-memory

File: Makefile.envs [hunk 1]
```diff
@@ -239,8 +240,9 @@ ifeq ($(DISABLE_DYLINK), 1)
 		-s USE_ZLIB \
 		-s USE_BZIP2 \
 		-s FORCE_FILESYSTEM=1 \
-		-s TOTAL_MEMORY=20971520 \
+		-s INITIAL_MEMORY=20971520 \
 		-s ALLOW_MEMORY_GROWTH=1 \
+                -s MAXIMUM_MEMORY=4GB \
```
Comment (hoodmane) [inline Makefile.envs:245]: Fix the white space on this line.

**Review by hoodmane — APPROVED** [review]
Thanks @airen1986!

File: Makefile.envs (hunk 1 above)
Comment (airen1986) [reply Makefile.envs:245]: Fixed it in commit 1e72ba7

Comment (hoodmane) [reply Makefile.envs:245]: The diff still shows 8 spaces instead of 2 tabs.

Comment (airen1986) [reply Makefile.envs:236]: Ok, I am confused, can someone help me on this?

**Comment by hoodmane** [comment]
Thanks @airen1986!

**Pull request merged**